sentences/el/vacuum_HassVacuumStart.yaml

```diff
@@ -0,0 +1,12 @@
+language: el
+intents:
+  HassVacuumStart:
+    data:
+      - sentences:
+          - "ξεκίνα <n>"
+          - "άρχισε <n>"
+          - "ξεκίνησε <n>"
+        requires_context:
+          domain: vacuum
```

Comment:
@synesthesiam this is no longer needed right, because it's already defined here? https://github.com/home-assistant/intents/blob/main/intents.yaml#L937-L939